### PR TITLE
feat:implemented wish function

### PR DESCRIPTION
**Pull request by SaiVasudha**

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -38,4 +38,8 @@ function mul(a,b){
  */
 function sub(a,b){
     return a-b;
+}
+
```
Comment (Padiprasannakumar123) [inline math.js:42]: gtggggg

File: math.js [hunk 2]
```diff
@@ -38,4 +38,8 @@ function mul(a,b){
...
+function  greet() {
+    print("hey guys!.. good Afternoon");
```
Comment (Padiprasannakumar123) [inline math.js]: use return statement

Comment (SaiVasudha) [reply math.js]: okk i will do that

**Review by Padiprasannakumar123 — APPROVED** [review]
LGTM

**Pull request merged**